.github/workflows/pr_test_cpu.yml

```diff
@@ -53,6 +51,17 @@ jobs:
     uses: ./.github/workflows/tests.yml
     with:
       os: ${{ matrix.os }}
-      python-version: '["3.7", "3.10"]'
+      python-version: '["3.10"]'
```

Comment:
for the nightly, i think we can just target the `dynamo` tests -- this way we  don't waste time seeing possible issues related to nightly on other parts of the code -- It should also decrease the test time on CI

Reply:
agree